ttyplot.1

```diff
@@ -83,6 +83,15 @@ Print the current version and exit.
 .It Fl h
 Print this help message and exit.
 .El
+.Sh KEY BINDINGS
+When reading data from a pipe,
+.Nm
+accepts the following commands typed at the terminal:
```

Comment:
I find the word "command" really unfortunate for a hotkey. Can we replace it also here?

Reply:
I checked a few man pages of full-screen programs, and the word “commands” seems quite standard, much more than “hotkeys”:

* less(1) – **COMMANDS** 
* htop(1) – **INTERACTIVE COMMANDS** The following commands are supported [...]
* picocom(1) – **COMMANDS** Commands are given to picocom by [...]
* tmux(1) – **DEFAULT KEY BINDINGS** tmux may be controlled [...] followed by a command key.